pupa/scrape/base.py

```diff
@@ -5,9 +5,10 @@
 from collections import defaultdict, OrderedDict
 
 import scrapelib
+from django.conf import settings
```

Comment:
Let me try to describe the issue a little better. The goal of this PR is to patch pupa settings used by the base scraper in the update cli module. That means the two modules have to import settings in the same way, either directly or via django. I noticed that settings are imported in different ways in different parts of pupa, and since django comes with a simple settings override context manager, I decided to use it instead of writing another context manager to do the same thing (which isn't hard, of course). It sounds like you all would prefer to import settings directly, which is fine with me. In that case, my only question is why *any* parts of pupa import settings via django.